extensions/wikia/AdEngine/js/lookup/prebid/priceParsingHelper.js

```diff
@@ -3,8 +3,9 @@ define('ext.wikia.adEngine.lookup.prebid.priceParsingHelper', [
 	'ext.wikia.adEngine.utils.sampler',
 	'wikia.geo',
 	'wikia.instantGlobals',
-	'wikia.log'
-], function (sampler, geo, instantGlobals, log) {
+	'wikia.log',
+	'wikia.window'
+], function (sampler, geo, instantGlobals, log, win) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 5 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)